graphql-dgs-spring-webmvc/src/main/kotlin/com/netflix/graphql/dgs/mvc/DgsRestController.kt

```diff
@@ -71,6 +72,11 @@ open class DgsRestController(
     open val mapper: ObjectMapper = jacksonObjectMapper()
 ) {
 
+    companion object {
```

Comment:
Isn't the Kotlin convention to put the companion object at the bottom?

Reply:
You're right, I guess it is, according to [this](https://kotlinlang.org/docs/coding-conventions.html#class-layout); guess it just feels weird to me to put constants / loggers at the end of a class.